subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/InstrumentationTaskConfigurator.kt

```diff
@@ -0,0 +1,8 @@
+package com.avito.instrumentation.internal
+
+import com.avito.instrumentation.InstrumentationTestsTask
+
+internal interface InstrumentationTaskConfigurator {
+
+    fun configure(task: InstrumentationTestsTask)
```

Comment:
```suggestion
    fun configure(task: Provider<InstrumentationTestsTask>)
```
Would be more Gradle way

Reply:
Or you guarantee laziness at the upper level?

Reply:
I couldn't find a way to lazily configure task at the moment of registration, like here:
https://github.com/avito-tech/avito-android/pull/1261/files#diff-a9f5557634124eb2a0ba2c4457392e08c1c8ead18a370ff37347d311470e210cR57

It wants Action<InstrumentationTestsTask> as parameter, which i use but only to iterate through `Configurators`
Not sure if passing knowledge that it is a configuration is worth it.